src/Symfony/Bundle/FrameworkBundle/Command/ConfigDebugCommand.php

```diff
@@ -104,6 +104,8 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                 $io->title(
                     \sprintf('Current configuration for %s', $name === $extensionAlias ? \sprintf('extension with alias "%s"', $extensionAlias) : \sprintf('"%s"', $name))
                 );
+
+                $io->comment(\sprintf('Documentation at %s', $this->getDocUrl($extension, $container)));
```

Comment:
is it possible that this returns no URL? then shouldn't we skip displaying the message when it happens?

Reply:
You're right, skip added